### PR TITLE
fix empty dropdown

### PR DESCRIPTION
**Pull request by Arhell**
issue https://github.com/yiisoft-contrib/yiiframework.com/issues/603

### CODE REVIEW

File: views/site/team.php [hunk 1]
```diff
@@ -34,7 +34,7 @@
                         <?php foreach($row as $i=>$member):?>
                             <div class="col-sm-4 col-md-2 person-card <?= ($i==0 && count($row)<6)?'col-md-offset-'.(6-count($row)):'' ?>">
                                     <div class="avatar">
-                                        <img src="<?= Html::encode(Yii::getAlias($member['photo'] ?? '@web/image/team/noimage.png')) ?>" class="img-responsive" alt="" />
+                                        <img src="<?= Html::encode(Yii::getAlias($member['photo'] ?? '@web/image/team/noimage.png')) ?>" class="img-responsive" alt="<?= Html::encode($member['name'])?>" />
```
Comment (samdark) [inline views/site/team.php:37]: As far as I remember, `alt` is read by screen readers so it will repeat name twice and that's not good.

Comment (Arhell) [reply views/site/team.php:37]: For example, there is also a description in alt
https://www.yiiframework.com/books

Comment (samdark) [reply views/site/team.php:37]: https://www.w3.org/WAI/tutorials/images/decision-tree/

Comment (Arhell) [reply views/site/team.php]: i update PR to fix this issue https://github.com/yiisoft-contrib/yiiframework.com/issues/599

File: assets/src/js/custom.js [hunk 2]
```diff
@@ -29,4 +29,8 @@ $(document).ready(function() {
       });
   });
 
+    if($(".version-selector .dropdown-menu").is(":empty")) {
+        $(".version-selector .dropdown-menu").hide();
+    }
```
Comment (samdark) [inline assets/src/js/custom.js:34]:
```suggestion
    $(".version-selector .dropdown-menu:empty").hide();
```

**Pull request merged**